src/js/pages/Index.js

```diff
@@ -20,6 +20,11 @@ import SidebarStore from "../stores/SidebarStore";
 import UserLanguageStore from "../stores/UserLanguageStore";
 import { hasViewportChanged, getCurrentViewport } from "../utils/ViewportUtil";
 import * as viewport from "../constants/Viewports";
+import {
+  compare,
+  getVersionFromVersionObject,
+  showNotification
```

Comment:
This function should live somewhere else. Not in the `UpdateStream`.

Reply:
You are right, Mike and I even talked about this yesterday.

Reply:
I will leave it there for now, until @mperrotti and I decide the best place to have it.